### PR TITLE
KBV-305 session lambda receive JAR

### PR DESCRIPTION
**Pull request by tal-nagra**
### What changed

The address CRI session lambda function now receives a JWT-secured authorisation request

### Issue tracking

- [KBV-305](https://govukverify.atlassian.net/browse/KBV-305)

### CODE REVIEW

File: lib/src/main/java/uk/gov/di/ipv/cri/address/library/domain/RawSessionRequest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package uk.gov.di.ipv.cri.address.library.domain;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+
+public class RawSessionRequest {
+    @JsonAlias("client_id")
```
Comment (KaraokeStu) [inline lib/src/main/java/uk/gov/di/ipv/cri/address/library/domain/RawSessionRequest.java:6]: Are we ever going to re-serialize this?
If so, we should use @JsonProperty instead

File: lib/src/main/java/uk/gov/di/ipv/cri/address/library/domain/RawSessionRequest.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    private String clientId;
+
+    @JsonAlias("request")
```
Comment (KaraokeStu) [inline lib/src/main/java/uk/gov/di/ipv/cri/address/library/domain/RawSessionRequest.java:9]: As above, if we're ever going to re-serialize

**Review by KaraokeStu — COMMENTED** [review]
Just my comments on re-serializing the object, if we're not going to ever reserialize it, then JsonAlias is fine

File: lib/src/main/java/uk/gov/di/ipv/cri/address/library/domain/RawSessionRequest.java (hunk 1 above)
Comment (tal-nagra) [reply lib/src/main/java/uk/gov/di/ipv/cri/address/library/domain/RawSessionRequest.java:6]: At present, there is no requirement to re-serialise

File: lib/src/main/java/uk/gov/di/ipv/cri/address/library/domain/RawSessionRequest.java (hunk 2 above)
Comment (tal-nagra) [reply lib/src/main/java/uk/gov/di/ipv/cri/address/library/domain/RawSessionRequest.java:9]: At present, there is no requirement to re-serialise

**Review by KaraokeStu — APPROVED** [review]

**Pull request merged**